x-pack/test/functional/apps/index_management/data_streams_tab/data_streams_tab.ts

```diff
@@ -88,6 +88,16 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       await testSubjects.click('closeDetailsButton');
     });
 
+    it('shows the correct index mode in the details flyout', async () => {
+      // Open details flyout
```

Comment:
It would be good to have tests that also check for logsdb

Reply:
Added with [2590ba4](https://github.com/elastic/kibana/pull/197874/commits/2590ba48f300a2686804ef60e4b25436658d2bfb).